docs/shared/overview/_payment/PaymentStartedAndFinished.mdx

```diff
@@ -0,0 +1,37 @@
+import * as IosBasketService from '@site/docs/ios/advanced/_shared/_miamManager/_basketService/';
+import * as AndroidBasketService from '@site/docs/android/advanced/_shared/_miamManager/_basketService/';
+
+
+export function PaymentStarted(props){
+    if(props.platform == "android") { return <AndroidBasketService.PaymentStarted  /> } else return  <IosBasketService.PaymentStarted  />
+}
+
+export function PaymentFinished(props){
+    if(props.platform == "android") { return <AndroidBasketService.PaymentFinished  /> } else return  <IosBasketService.PaymentFinished  />
+}
+
+
+There are two functions that need to be called during the checkout experience.
+Mealz does not collect payment, but we must reset our basket & lodge the transaction to share with your commerical team.
+Checkout is a crucial part of the Mealz experience.
+
+### Payment Started
+
+
+When your use selects "Buy now", or when the payment tunnel is begun, you must call `paymentStarted.`
+
+<PaymentStarted platform ={props.platform}/>
+
+This will save the payment information on our end & allow us to cross reference with your team for every transaction Mealz is involved with.
+Internally, we do NOT track payments where the value of products added by Mealz is 0, so you do not need to implement this logic.
+
+### Payment Finished
+
+When your payment tunnel is over, & you've returned a "Payment Success", you must call `paymentFinished.`
+
+:::danger
+It is of crucial importance that you pass the correct values for the totalAmount & totalProductCount.
+Please ensure these are the values of the transaction, NOT the values after the transaction (probably an empty basket)
```

Comment:
Do it before reseting your basket otherwith we will consider missing products as removed products